use-cases/in-memory-data-grid.html

```diff
@@ -58,60 +57,60 @@ <h1><strong>In-Memory Data Grid</strong> with Apache Ignite</h1>
     <div class="container">
 
         <p>
-            Apache Ignite® is an in-memory data grid that accelerates and scales your databases, services, and APIs.
-            It supports key-value and ANSI SQL APIs, ACID transactions, co-located compute, and machine learning
-            libraries required for real-time applications.
-        </p>
-        <p>
-            An in-memory data grid deployment is a read-through/write-through caching strategy, in which the application
-            layer treats the data grid as the primary data store. The application layer writes to and reads from Ignite.
-            Ignite ensures that any underlying database stays updated and consistent with the in-memory data.
+            The Apache Ignite® in-memory data grid accelerates and scales your databases, services, 
+            and APIs. It supports key-value and ANSI SQL APIs, ACID transactions, co-located processing, 
+            and machine learning libraries. Ignite is frequently used to increase the performance and 
+            scalability of real-time applications, as a <a href="/use-cases/digital-integration-hub.html">digital integration hub</a> to provide real-time data 
+            access to one or many applications to data from one or many siloed data sources, for <a href="/use-cases/high-performance-computing.html">high 
```

Comment:
I do see that we want to get more searches of DIH and "cache" terms by adding them to the first paragraph here but that's a tangential topic and we shouldn't overwhelm our readers with more terms, especially, at the beginning of the page. 

I'll rework this paragraph after the merge.